README.md

```diff
@@ -1,46 +1,41 @@
-# The Bisq website
+# Bisq Network Website
 
-To run the Jekyll site locally:
+This repository hosts the source code for the Jekyll website hosted at https://bisq.network.
 
-0. Check Ruby verion `$ ruby -v` (needs to be 2.2.5 or higher) https://www.moncefbelyamani.com/how-to-install-xcode-homebrew-git-rvm-ruby-on-mac/
-1. If needs update run `$ rvm install 2.5.1`)
-2. Install bundler per instructions at http://bundler.io  `gem install bundler`
-3. Clone and cd into this repository
-4. Run `$ bundle install`
-5. Run `$ bundle exec jekyll serve`
-6. Browse the site at http://127.0.0.1:4000/
+## Contributing
+
+Please submit proposed changes as pull requests, ideally with commit messages that follow [these conventions](https://github.com/bisq-network/style/issues/9).
+
+A pull requests should generally have a singular, well-defined purpose. For example, a pull request to add a blog post should just add a blog post, not add a blog post _and_ change spacing/indentation on some other page.
 
-Note that LiveReload functionality is enabled by default in `_config.yml`.
-This means that if you have the LiveReload browser extension installed,
-you'll see local changes reload into the browser automatically on save.
+You should fork this repository, create a new branch, and and make all your changes there. Please don't commit your changes to the `master` branch.
 
+**Please thoroughly test pull requests before submitting.**
 
-Dev running in 0.0.0.0 (port 4000 closed don't know why)
+For smaller changes, it's usually enough to build and observe the site locally. Don't forget that the site is responsive, so testing on mobile and desktop screens is important, as is testing on multiple browsers.
 
-1. Run `$ jekyll serve --port 9999 --host=0.0.0.0`
-2. Go to `http://port-4000.bisq-website-pedromvpg.codeanyapp.com/` for port 4000
+It's also not a bad idea to do a `diff` on the source code before submitting (for example, to make sure stray characters weren't added somewhere by mistake). You can check for changes by running `git diff <latest upstream commit id> <your branch's latest commit id>` or by using a graphical tool like Meld.
 
+For changes that may alter the site's functionality, it can be helpful to `diff` the built code base in `_site/` against the existing built code base. You can do this by running something like `git diff -ru <upstream _site folder> <local _site folder>` or by comparing the 2 directories with a graphical tool like Meld.
```

Comment:
```suggestion
For changes that may alter the site's functionality, it can be helpful to `diff` the generated HTML code in `_site/` against the current live production website's HTML code. You can do this by building `master` branch and your work branch in separate folders, and then run something like `git diff -ru <upstream/master _site folder> <local/work _site folder>` or by comparing the 2 directories with a graphical tool like Meld.
```